sdk/textanalytics/azure-ai-textanalytics/samples/sample_get_detailed_diagnostics_information.py

```diff
@@ -45,13 +45,17 @@ def get_detailed_diagnostics_information(self):
             "L'hôtel n'était pas très confortable. L'éclairage était trop sombre."
         ]
 
+        json_responses = []
+
         def callback(resp):
             _LOGGER.debug("document_count: {}".format(resp.statistics["document_count"]))
             _LOGGER.debug("valid_document_count: {}".format(resp.statistics["valid_document_count"]))
             _LOGGER.debug("erroneous_document_count: {}".format(resp.statistics["erroneous_document_count"]))
             _LOGGER.debug("transaction_count: {}".format(resp.statistics["transaction_count"]))
             _LOGGER.debug("model_version: {}".format(resp.model_version))
-            _LOGGER.debug("raw_response in json format: {}".format(json.dumps(resp.raw_response)))
+            _LOGGER.debug("raw_response: {}".format(resp.raw_response))
```

Comment:
we're still logging it here

Reply:
I still logged the raw_response since it's kind of different, but I see where you're coming from, will remove it so we don't have semi-duplicate outputs